subprojects/core/src/testFixtures/groovy/org/gradle/api/tasks/AbstractSpockTaskTest.groovy

```diff
@@ -159,61 +156,48 @@ public abstract class AbstractSpockTaskTest extends AbstractProjectBuilderSpec {
         thrown(InvalidUserDataException)
     }
 
-    def testExecuteDelegatesToTaskExecuter() {
-        final AbstractTask task = getTask()
-        TaskExecuter executer = Mock()
-        task.setExecuter(executer);
-
-        when:
-        task.execute()
-
-        then:
-        1 * executer.execute(task, _ as TaskStateInternal, _ as TaskExecutionContext)
-
-    }
```

Comment:
Why remove this test?

Reply:
I added it again. I removed it since I thought the test was pretty pointless. I hope no one set the executer on `Task` to their own implementation.